src/pages/Addons/components/registry-manage/index.tsx

```diff
@@ -294,7 +321,28 @@ class RegistryManageDialog extends React.Component<Props, State> {
                     />
                   </Form.Item>
                 </Col>
-                <If condition={selectType === 'Github' || selectType === 'Gitee'}>
+                <If condition={selectType === 'Gitlab'}>
+                  {' '}
+                  <Col span={3} style={{ padding: '8px' }}>
+                    <Form.Item label={<Translation>Repository name</Translation>} required>
```

Comment:
What is the repository name? the URL does not include the repository name? advice to add a help message.

Reply:
Because the owner level in the gitlab repository address is not fixed, gitlab must determine one of the owner and repo to ensure correct parsing

For example, gitlab supports oam/beijing/velaux and oam/velaux repository paths,

To get the project ID of gitlab, you must determine one of the owner and repo

For help information, do I think the repository name is enough

Or do you have a better idea

Reply:
I have added the help message for repository name